<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>almox_gestaoColab</class>
 <widget class="QDialog" name="almox_gestaoColab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gerenciar Colaboradores</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>661</width>
     <height>481</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_novoProduto">
    <attribute name="title">
     <string>Novo Colaborador</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_username">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Nome Colaborador:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_nome">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>491</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_acesso">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>A</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>C</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Acesso:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>70</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Senha:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_senha">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>70</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_gravarColab">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>380</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#288;
color:#fff;
border-radius:10px;
font-size:10pt;</string>
     </property>
     <property name="text">
      <string>Gravar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_limpar">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>380</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#288;
color:#fff;
border-radius:10px;
font-size:10pt;</string>
     </property>
     <property name="text">
      <string>Limpar</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_gestaoProdutos">
    <attribute name="title">
     <string>Gestão Colaboradores</string>
    </attribute>
    <widget class="QTableWidget" name="tw_gc_colab">
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>31</y>
       <width>341</width>
       <height>421</height>
      </rect>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>271</width>
       <height>361</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>Arial Black</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Código Colaborador</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_gc_codigo">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>Arial Black</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Nome Colaborador</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_gc_nome"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <family>Arial Black</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Username Colaborador</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_gc_username"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <family>Arial Black</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Senha Colaborador</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_gc_senha"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>Arial Black</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Acesso</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_acesso">
          <property name="font">
           <font>
            <family>Arial Black</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <item>
           <property name="text">
            <string>A</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>B</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>C</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btn_gc_gravar">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>421</y>
       <width>56</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#288;
color:#fff;
border-radius:10px;
font-size:10pt;</string>
     </property>
     <property name="text">
      <string>Salvar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_gc_excluir">
     <property name="geometry">
      <rect>
       <x>142</x>
       <y>421</y>
       <width>62</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#288;
color:#fff;
border-radius:10px;
font-size:10pt;</string>
     </property>
     <property name="text">
      <string>Excluir</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
